kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -233,6 +233,10 @@ func (options *SubmitJobOptions) Validate() error {
 			return fmt.Errorf("Submission mode of the Ray Job must be set to 'InteractiveMode'")
 		}
 
+		if options.RayJob.Spec.Entrypoint != "" {
+			log.Printf("entrypoint in Ray job is set to '%s', but it will be overwritten by the command's entrypoint '%s'", options.RayJob.Spec.Entrypoint, options.entryPoint)
```

Comment:
context is https://github.com/ray-project/kuberay/pull/3186

Reply:
I see. Is it still useful to have this warning to prevent confusion?